Automattic/woocommerce-payments
# Expand Stripe Link test coverage

Fixes https://github.com/Automattic/woocommerce-payments/issues/5338

#### Changes proposed in this Pull Request
This PR expands the E2E coverage for Stripe Link-related functionality. The flow now runs the shortcode checkout with UPE/split UPE, and the blocks checkout tests Link flow with UPE enabled. 

One more unit test has been added to cover the implementation in `class-wc-payments.php`.









#### Testing instructions
1. Run e2e tests locally and ensure they are green.
2. Re-run the GH Actions flow for this PR and ensure the tests are green. 





-------------------


**Post merge**
